templates/docs/document.html

```diff
@@ -18,68 +52,250 @@
       </form>
     </div>
   </section>
+
   <div class="l-docs">
-    <aside class="l-docs-sidebar" id="navigation">
-      <i class="p-sidenav__toggle p-icon--menu u-hide u-show--small"></i>
-      <nav class="p-sidenav__body u-hide--small">
-        {{ navigation | safe }}
+    <aside class="col-3 l-docs-sidebar">
+      {% if versions | length > 1 %}
+      <label for="version-select" class="u-hide">Version</label>
+      <select name="version-select" id="version-select" onChange="window.location.href=this.value">
+      {% for version in versions %}
+        {% set active = docs_version == version['path'] %}
+        <option value="{{ version_paths[version['path']] }}"{% if active %} selected{% endif %}>{{ version['version'] }}</option>
+      {% endfor %}
+      <select>
+      {% endif %}
+
+      <div class="p-strip is-shallow">
```

Comment:
This looks great @sparkiegeek The only thing I spotted is that the toggle navigation is hard to the left. I believe you should wrap the contents of this strip with a `<div class="row">`